Quadriga/src/main/webapp/WEB-INF/views/public/sitesnavbar.jsp

```diff
@@ -0,0 +1,4 @@
+ <li style="color:red"> <h4> <b> Quadriga Network Projects </b> </h4></li>
```

Comment:
you won't this file then. also, this should not be red (just have it look like all the other project sites) (it was just red because it was a crappy mockup ;)